src/lib/opts.rs

```diff
@@ -79,19 +78,19 @@ For support please contact: care@singulargenomics.com
 #[derive(Parser, Debug, Clone)]
 #[clap(name = TOOL_NAME, version = built_info::VERSION.as_str(), about=SHORT_USAGE, long_about=LONG_USAGE, term_width=0)]
 pub struct Opts {
-    /// Path to the input FASTQs.
+    /// Path to the input FASTQs, or path prefix if not a file.
     #[clap(long, short = 'f', display_order = 1, required = true, multiple_values = true)]
     pub fastqs: Vec<PathBuf>,
 
     /// Path to the sample metadata.
     #[structopt(long, short = 's', display_order = 2)]
     pub sample_metadata: PathBuf,
 
-    /// Read structures, one per input FASTQ.
+    /// Read structures, one per input FASTQ. Do not provide when using a path prefix for FASTQs.
     #[clap(long, short = 'r', display_order = 3, required = true, multiple_values = true)]
```

Comment:
Still has required=true.  Probably need to drop that and separately validate that it's given _if_ a prefix is given?